mypy/test/data/check-statements.test

```diff
@@ -618,6 +620,36 @@ except exc as e: pass             # E: Exception type must be derived from BaseE
 except BaseException() as b: pass # E: Exception type must be derived from BaseException
 [builtins fixtures/exception.py]
 
+[case testTupleValueAsExceptionType]
+import typing
+def exc() -> BaseException: pass
+class E1(BaseException): pass
+class E2(E1): pass
+class E3(E1): pass
+
+exs1 = (E1, E2)
+try: pass
+except exs1 as e1:
+    reveal_type(e1) # E: Revealed type is '__main__.E1'
+
+exs3 = (E2, E3)
+try: pass
+except exs3 as e3:
+    reveal_type(e3) # E: Revealed type is '__main__.E1'
```

Comment:
`E1` is the nearest common ancestor to `E2` and `E3`, so it is the most specific type that generalizes both.